From bba0e878151f8955f49ade482beb335aa8bf4031 Mon Sep 17 00:00:00 2001
From: "Arnaud Patard (Rtp)" <arnaud.patard@rtp-net.org>
Date: Fri, 25 Sep 2020 20:36:58 +0200
Subject: [PATCH 4/9] RFC: drivers/video/rockchip/rk_edp.c: Change clock rate

The current code is setting the clock rate to 192000000, but
due to the current device-tree configuration and linux code,
it should rather be 100000000.

Signed-off-by: Arnaud Patard <arnaud.patard@rtp-net.org>
---
 drivers/video/rockchip/rk_edp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/rockchip/rk_edp.c b/drivers/video/rockchip/rk_edp.c
index 13fa78aced..c2dc79faac 100644
--- a/drivers/video/rockchip/rk_edp.c
+++ b/drivers/video/rockchip/rk_edp.c
@@ -1078,7 +1078,7 @@ static int rk_edp_probe(struct udevice *dev)
 	}
 	ret = clk_get_by_index(uc_plat->src_dev, 0, &clk);
 	if (ret >= 0) {
-		ret = clk_set_rate(&clk, 192000000);
+		ret = clk_set_rate(&clk, 100000000);
 		clk_free(&clk);
 	}
 	if (ret < 0) {
-- 
2.28.0

